src/SFA.DAS.RoATPService.Application/Validators/OrganisationValidator.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace SFA.DAS.RoATPService.Application.Validators
+{
+    using System;
+
+    public class OrganisationValidator : IOrganisationValidator
+    {
+        public bool IsValidOrganisationId(Guid organisationId)
+        {
+            if (organisationId == null || organisationId == Guid.Empty)
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        public bool IsValidApplicationRouteId(int applicationRouteId)
+        {
+            return (applicationRouteId > 0);
+        }
+
+        public bool IsValidUKPRN(long ukPrn)
+        {
+            return (ukPrn >= 10000000 && ukPrn <= 99999999);
+        }
+
+        public bool IsValidLegalName(string legalName)
+        {
+            if (String.IsNullOrWhiteSpace(legalName))
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        public bool IsValidStatusDate(DateTime statusDate)
+        {
+            return (statusDate > DateTime.MinValue);
+        }
+
+        public bool IsValidStatus(string status)
+        {
+            return !String.IsNullOrWhiteSpace(status);
+        }
+    }
```

Comment:
More an FYI than a comment here, I'm currently putting in a validationservice within assessor-service to validate common stuff, like email, ukprn, organisationId etc.  It may be we branch these off into it's own repo library, not sure.  The plan is to have a single point for testing common validations.   I'll keep you posted.

Reply:
That sounds good, I think a separate repo would be a good idea as we have quite a bit of shared validation between RoATP and EPAO

Reply:
Story ON-1436 has the details